flytekit/core/data_persistence.py

```diff
@@ -61,6 +61,41 @@ def s3_setup_args(s3_cfg: configuration.S3Config, anonymous: bool = False):
     return kwargs
 
 
+def azure_setup_args(azure_cfg: configuration.AzureBlobStorageConfig, anonymous: bool = False):
+    kwargs: Dict[str, Any] = {}
+
+    if azure_cfg.account_name:
+        kwargs["account_name"] = azure_cfg.account_name
+    if azure_cfg.account_key:
+        kwargs["account_key"] = azure_cfg.account_key
+    if azure_cfg.client_id:
+        kwargs["client_id"] = azure_cfg.client_id
+    if azure_cfg.client_secret:
+        kwargs["client_secret"] = azure_cfg.client_secret
+    if azure_cfg.tenant_id:
+        kwargs["tenant_id"] = azure_cfg.tenant_id
+    kwargs[_ANON] = anonymous
+    return kwargs
+
+
+def get_fsspec_storage_options(
+    protocol: str, data_config: typing.Optional[DataConfig] = None, anonymous: bool = False, **kwargs
+) -> typing.Dict:
```

Comment:
is this always [str, str]?

Reply:
Its `Dict[str, Any]`. I made it more precise.